api/Element.json

```diff
@@ -5431,22 +5431,22 @@
               "version_added": null
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "≤14"
             },
             "safari": {
-              "version_added": null
+              "version_added": "3"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "1"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": null
+              "version_added": "4.4.3"
```

Comment:
Maybe <=37 if this isn't based on testing?

Reply:
Whoops, meant to change this to 1!  (I haven't tested it, but I'm pretty confident this would be in Android 1 based upon the engine versions in relation to Safari 3.)